### PR TITLE
Fix/expired verify email

### LINKED ISSUE #191: Invalid or expired email verification links are not handled properly
If a user clicks on an email verification link that is either expired or invalid he / she sees a success screen, even if an error is thrown. We should handle the error properly and display a message accordingly!

I believe the line of code where we don't handle a possible exception is `CompleteOfferHelp.jsx:32`

```js
await fb.auth.signInWithEmailLink(email, window.location.href);
```

### PR DESCRIPTION
**Pull request by Maurice22**
**What changes does this PR introduce**

On the complete offer help we now first show a loading icon while validating the email link. Once it is checked, it shows either a success or error message.

Closes #191

### CODE REVIEW

**Comment by florianschmidt1994** [comment]
If you just open the url `/#/complete-offer-help` it will show the spinner forever

File: src/views/CompleteOfferHelp.jsx [hunk 1]
```diff
@@ -7,10 +7,45 @@ import { useTranslation } from 'react-i18next';
...
+      uid: firebase.auth().currentUser.uid,
+      timestamp: Date.now(),
+      coordinates: new fb.app.firestore.GeoPoint(lat, lng),
+    });
+
+    fb.analytics.logEvent('success_subscribe_region_confirmation');
+  };
 
   useEffect(() => {
     async function completeSignup() {
```
Comment (florianschmidt1994) [inline src/views/CompleteOfferHelp.jsx:51]: Can we refactor this to use an early return with a !

**Comment by florianschmidt1994** [comment]
Looks good to me. Can you fix the merge conflicts?

**Comment by Maurice22** [comment]
There we go

**Pull request merged**